purchase_order_approved/views/purchase_order_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form - purchase_order_approved</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <button name="button_draft" position="attributes">
+                <attribute name="states">cancel,to approve,approved</attribute>
+            </button>
+            <button name="button_approve" position="after">
+                <button name="button_release" type="object" states='approved'
```

Comment:
@lreficent Is there any security mechanism (group) here (or is this module is just to have one more step) ?

Reply:
the goal is just the extra step.